src/rebar_port_compiler.erl

```diff
@@ -568,9 +568,21 @@ erl_interface_dir(Subdir) ->
     end.
 
 default_env() ->
+    Arch = os:getenv("REBAR_TARGET_ARCH"),
+    Vsn = os:getenv("REBAR_TARGET_ARCH_VSN"),
     [
-     {"CC" , "cc"},
-     {"CXX", "c++"},
+     {"CC" , get_tool(Arch,Vsn,"gcc","cc")},
+     {"CXX", get_tool(Arch,Vsn,"g++","c++")},
+     {"AR" , get_tool(Arch,"ar","ar")},
```

Comment:
Maybe its' the right thing to do, but shouldn't we consider other toolchains than gcc as well? Would an extra env var be a good solution?